src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -150,6 +150,30 @@ public IPAddress(byte[] address, long scopeid)
             PrivateScopeId = (uint)scopeid;
         }
```

Comment:
Can the IPAddress(byte[], long) ctor be implemented in terms of this new one?
```C#
public IPAddress(byte[] address, long scopeid) :
    this(new ReadOnlySpan<byte>(address ?? new ArgumentNullException(nameof(address))), scopeid)
{
}
```
or does that negatively impact the perf of the byte[]-based ctor?